SDL/Quintuplet.cu

```diff
@@ -428,6 +428,236 @@ __device__ bool SDL::runQuintupletDefaultAlgo(struct SDL::modules& modulesInGPU,
+        mdsInGPU.anchorPhi[thirdMDIndex],       // outer (hit 4) t3_0_phi
+        mdsInGPU.anchorZ[thirdMDIndex],         // outer (hit 3) t3_0_z
+        sqrtf(x3*x3 + y3*y3),                   // outer (hit 3) t3_0_r
+        float(modulesInGPU.layers[lowerModuleIndex3] + 6*is_endcap3), // outer (hit 3) t3_0_layer
+        mdsInGPU.anchorEta[fourthMDIndex],      // outer (hit 4) t3_2_eta
+        mdsInGPU.anchorPhi[fourthMDIndex],      // outer (hit 4) t3_2_phi
+        mdsInGPU.anchorZ[fourthMDIndex],        // outer (hit 4) t3_2_z
+        sqrtf(x4*x4 + y4*y4),                   // outer (hit 4) t3_2_r
+        float(modulesInGPU.layers[lowerModuleIndex4] + 6*is_endcap4), // outer (hit 4) t3_2_layer
+        mdsInGPU.anchorEta[fifthMDIndex],       // outer (hit 5) t3_4_eta
+        mdsInGPU.anchorPhi[fifthMDIndex],       // outer (hit 5) t3_4_phi
+        mdsInGPU.anchorZ[fifthMDIndex],         // outer (hit 5) t3_4_z
+        sqrtf(x5*x5 + y5*y5),                   // outer (hit 5) t3_4_r
+        float(modulesInGPU.layers[lowerModuleIndex5] + 6*is_endcap5), // outer (hit 5) t3_4_layer
+        log10((innerRadius+outerRadius)*3.8f*1.602f/(2*100*5.39f)), // t5_pt
+        /* log10(2.99792458e-3f*3.8f*innerRadius), // t5_pt // BAD TEST */
+        mdsInGPU.anchorEta[segmentsInGPU.mdIndices[2*tripletsInGPU.segmentIndices[2*innerTripletIndex+layer2_adjustment]]], // t5_eta
+        mdsInGPU.anchorPhi[segmentsInGPU.mdIndices[2*tripletsInGPU.segmentIndices[2*innerTripletIndex+layer2_adjustment]]], // t5_phi
+        log10(innerRadius),                            // t5_innerRadius
+        log10(bridgeRadius),                           // t5_bridgeRadius
+        log10(outerRadius)                             // t5_outerRadius"
+    };
+
+    // (0): Linear(in_features=38, out_features=32, bias=True) => x = x*W_T + b
+    float bias_0[32] = {
+        -0.453155100345611572265625000000,-3.414395570755004882812500000000, 0.461477220058441162109375000000,-0.154097318649291992187500000000,-0.470577239990234375000000000000, 2.228851079940795898437500000000,-0.163815096020698547363281250000,-2.296737670898437500000000000000,-0.098085217177867889404296875000, 2.589857339859008789062500000000, 2.105398178100585937500000000000,-0.525056004524230957031250000000, 1.790417790412902832031250000000,-2.162935256958007812500000000000, 0.012643844820559024810791015625,-1.595005750656127929687500000000, 0.222240477800369262695312500000,-3.097779273986816406250000000000,-0.748212277889251708984375000000, 1.027869462966918945312500000000,-0.504925668239593505859375000000, 0.079979300498962402343750000000,-0.680445253849029541015625000000, 1.331984400749206542968750000000, 2.027832508087158203125000000000,-0.063643321394920349121093750000,-0.304876327514648437500000000000,-0.020376743748784065246582031250, 2.847964286804199218750000000000,-1.884062528610229492187500000000, 2.168398141860961914062500000000,-2.673130989074707031250000000000
```

Comment:
for longer term, we should see how to load these from a file.
For this PR, I'd suggest to format the values in float precision to have a more compact code.

For the hackathon, I'm curious how much of this can be sped up by using half-precision and/or tensor operations